### DISCUSSION TITLE
<title>react image slider with map

### DISCUSSION THREAD

**Discussion by YaelOushky** [community/community#65595] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

som now abut good react image slider with map ?

**Comment by ShonChassel** [comment]
yes chek this 
https://www.npmjs.com/package/react-slideshow-image

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
